lib/models/Veeam.ts

```diff
@@ -72,9 +78,9 @@ export class VeeamCapacityInfo {
 
     static toBigInt(capacity: VeeamSOSApiSchema['CapacityInfo']): VeeamSOSApiSchema['CapacityInfo'] {
         return {
-            Capacity: BigInt(capacity?.Capacity || 0),
-            Available: BigInt(capacity?.Available || 0),
-            Used: BigInt(capacity?.Used || 0),
+            Capacity: BigInt(capacity?.Capacity.toString() || 0),
+            Available: BigInt(capacity?.Available.toString() || 0),
+            Used: BigInt(capacity?.Used.toString() || 0),
```

Comment:
why do we _need_ to convert to string?
- if will actually crash if capacity were null/undefined (`capacity?.`)
- according to the type (and typescript compiler), Capacity/Available/Used can now only be bigint already...

We can keep the `BigInt` call for extra caution, but we should avoid the systematic double conversion to and from string...